### PR TITLE
Feat/Update CoinAlpha connectors fees schema

### PR DESCRIPTION
**Pull request by mhrvth**
**Before submitting this PR, please make sure**:

- [x] Your code builds clean without any errors or warnings
- [x] You are using approved title ("feat/", "fix/", "docs/", "refactor/")

**A description of the changes proposed in the pull request**:

Replacement of a legacy definition of trade fee schemas with `TradeFeeSchema` in connectors maintained by CoinAlpha

**Tests performed by the developer**:

Ran unit tests

### CODE REVIEW

**Comment by mhrvth** [comment]
[sc-23435]

File: hummingbot/connector/exchange/okex/okex_utils.py [hunk 1]
```diff
@@ -1,16 +1,19 @@
...
+
 from hummingbot.client.config.config_var import ConfigVar
 from hummingbot.client.config.config_methods import using_exchange
-import zlib
+from hummingbot.core.data_type.trade_fee import TradeFeeSchema
 
-CENTRALIZED = True
+DEFAULT_FEES = TradeFeeSchema(
+    maker_percent_fee_decimal=Decimal("0.001"),
+    taker_percent_fee_decimal=Decimal("0.0015"),
```
Comment (petioptrv) [inline hummingbot/connector/exchange/okex/okex_utils.py:10]: Looks like these have changed too: https://www.okx.com/fees.html

**Review by petioptrv — CHANGES_REQUESTED** [review]

**Review by petioptrv — APPROVED** [review]
LGTM

**Comment by Deepali6997** [comment]
ASCENDEX

For some reason when we’re trying to run pure-market strategy in the PR214 branch, the strategy is not starting. Although the crypto assets are enough for a trade to take place. Also, didn't spot any error messages too. 

Screenshot 2022-04-21 at 8 29 46 PM
Screenshot 2022-04-21 at 8 30 45 PM
Screenshot 2022-04-21 at 8 30 03 PM

[logs_ascendex.log](https://github.com/CoinAlpha/hummingbot/files/8532150/logs_ascendex.log)


Same is happening in MASTER branch. 

Screenshot 2022-04-21 at 8 49 40 PM
Screenshot 2022-04-21 at 8 50 32 PM

[logs_ascendex_master.log](https://github.com/CoinAlpha/hummingbot/files/8532286/logs_ascendex_master.log)

BYBIT PERPETUAL

for bybit perpetual market, we can see that in the logs file the fees calculated is having a high precision. Hence, the value is not matching the fee deduction value shown on the exchange.

Also, the price is showing the same behaviour. 

Screenshot 2022-04-21 at 8 54 55 PM

Screenshot 2022-04-21 at 8 54 37 PM

Screenshot 2022-04-21 at 8 56 26 PM

[trades_perpetual_mm.csv](https://github.com/CoinAlpha/hummingbot/files/8532775/trades_perpetual_mm.csv)


OkEx

The maker fee is not shown in the .csv file . PFB the attached snaps from csv and exchange 

Screenshot 2022-04-21 at 10 00 12 PM

Screenshot 2022-04-21 at 10 00 45 PM



[trades_okEx.csv](https://github.com/CoinAlpha/hummingbot/files/8532773/trades_okEx.csv)

**Comment by Deepali6997** [comment]
LGTM 

- Created Ppm/perpetual strategy 
- Set a tight bid spread and wide ask spread
- Post fill, run history verbose and compared fees between exchange and .csv file
- Re-checking balance and confirming that fee is deducted from order size
- Set a tight bid spread and wide ask spread
- Post fill, run history verbose and compared fees between exchange and .csv file

All exchanges are behaving as expected

**Review by Deepali6997 — APPROVED** [review]
Test Summary:-

- Created Ppm/perpetual strategy
- Set a tight bid spread and wide ask spread 
- Post fill, ran history verbose and compared fees between exchange and .csv file
- Re-checking balance and confirming that fee is deducted from order size
- Set a tight ask spread and wide bid spread
- Post fill, ran history verbose and compared fees between exchange and .csv file
- Can confirm that following connectors are behaving as expected
( Binance spot , Binance_perpetual, ascendex, bybit perpetual , gate_io , huobi, kucoin, okex, probit)

**Comment by mhrvth** [comment]
Created a PR in the Foundation repo: https://github.com/hummingbot/hummingbot/pull/5247

**Pull request closed without merging**